### PR TITLE
Fix layout of Quick Start table view header

### PR DESCRIPTION
**Pull request by nheagy**
Fixes the layout issues mentioned [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/11028#issuecomment-464654733)

image

image

To test:
- View a site with Quick Start enabled, or add `return true` in `shouldShowQuickStartChecklist()`
- If you're Sylvester: Ensure the heading matches desired layout
- Otherwise: ensure the app still functions and the code is good

Update release notes:

- [ ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

**Comment by SylvesterWilmott** [comment]
Looks good @nheagy , thanks!

**Comment by SylvesterWilmott** [comment]
One thing I noticed in another PR was that the separator between the cells align to the icon. Can we just change this to it aligns with the text? This is just to remain consistent with the rest of the options on the site dashboard.

screenshot 2019-02-20 at 16 05 46

**Comment by nheagy** [comment]
@SylvesterWilmott Not sure which PR that was, but it may have been branched before that was changed. This is the latest appearance:

image

File: WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m [hunk 1]
```diff
@@ -35,7 +35,7 @@
 NSInteger const BlogDetailHeaderViewVerticalMargin = 18;
 CGFloat const BlogDetailGridiconAccessorySize = 17.0;
 CGFloat const BlogDetailBottomPaddingForQuickStartNotices = 80.0;
-CGFloat const BlogDetailQuickStartSectionHeight = 26.0;
+CGFloat const BlogDetailQuickStartSectionHeight = 35.0;
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m]: Shouldn't this be 36? 🤔

Comment (nheagy) [reply WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m]: There's an extra pixel coming from somewhere. This is the value that makes the layout match the mockups `¯\_(ツ)_/¯`

**Comment by nheagy** [comment]
@frosty I fixed the gap problem—thanks for catching that. It seems that implementing `tableView:viewForHeaderInSection:` changes this behaviour.

Also, a height of `35.0` makes things look right. I wasn't going to question it 😉

**Comment by nheagy** [comment]
Thanks @frosty and @SylvesterWilmott!

**Pull request merged**